h2/src/test/org/h2/test/scripts/aliases.sql

```diff
@@ -0,0 +1,82 @@
+-- Copyright 2004-2018 H2 Group. Multiple-Licensed under the MPL 2.0,
+-- and the EPL 1.0 (http://h2database.com/html/license.html).
+-- Initial Developer: H2 Group
+--
```

Comment:
please rename this file to something like
   derived_column_names.sql
since in our rather weird dialect of SQL, an ALIAS is a stored procedure

Reply:
I renamed it.

This feature is a standard feature, isn't it?
https://groups.google.com/forum/#!topic/h2-database/1StLJp-mxpc

Unfortunately, I don't have a recent versions of SQL standard. I can only say that this is supported by PostgreSQL and in SQL Server, but is not supported by Oracle and MySQL.

Reply:
if @lukaseder asked for it,  then it is pretty much guaranteed to be a standard SQL feature :-)

Reply:
Yes it is. Here's the ticket I created at the time. Seems to be a duplicate now: https://github.com/h2database/h2database/issues/288